### PR TITLE
Ignore protobuf

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,6 +29,8 @@ option(WITH_OFFLINE "Build offline interface?" OFF)
 option(USE_DEPTH_COMPUTE_STUBS "Use empty generated libs?" OFF)
 option(USE_DEPTH_COMPUTE_OPENSOURCE "Use an open source implementation?" OFF)
 option(WITH_GLOG_DEPENDENCY "Build with GLOG dependency to be used for logging" ON)
+option(WITH_PROTOBUF_DEPENDENCY "Build with GLOG dependency to be used for logging" ON)
```
Comment (dNechita) [inline CMakeLists.txt:32]: Remove glog word. Use protobuf

File: CMakeLists.txt [hunk 2]
```diff
@@ -29,6 +29,8 @@ option(WITH_OFFLINE "Build offline interface?" OFF)
 option(USE_DEPTH_COMPUTE_STUBS "Use empty generated libs?" OFF)
 option(USE_DEPTH_COMPUTE_OPENSOURCE "Use an open source implementation?" OFF)
 option(WITH_GLOG_DEPENDENCY "Build with GLOG dependency to be used for logging" ON)
```
Comment (dNechita) [inline CMakeLists.txt:31]: There is a readme where we explain in more detail these options. Add these new options as well.

File: CMakeLists.txt [hunk 3]
```diff
@@ -98,6 +100,12 @@ if(WITH_GLOG_DEPENDENCY)
         add_definitions(-DUSE_GLOG)
 endif()
 
+if(NOT WITH_PROTOBUF_DEPENDENCY AND NOT NXP)
```
Comment (dNechita) [inline CMakeLists.txt:103]: It's not just NXP that we support

**Review by dNechita — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (SeptimiuVana) [reply CMakeLists.txt]: Resolved.

File: CMakeLists.txt (hunk 3 above)
Comment (SeptimiuVana) [reply CMakeLists.txt]: Sorry, forgot about that. Fixed.

File: CMakeLists.txt (hunk 2 above)
Comment (SeptimiuVana) [reply CMakeLists.txt:31]: Updated.

**Review by dNechita — APPROVED** [review]

**Pull request merged**